pkg/apis/configuration/v1/kongingress_types.go

```diff
@@ -153,7 +153,7 @@ type KongIngressUpstream struct {
 	Healthchecks *kong.Healthcheck `json:"healthchecks,omitempty" yaml:"healthchecks,omitempty"`
 
 	// HashOn defines what to use as hashing input.
-	// Accepted values are: "none", "consumer", "ip", "header", "cookie".
+	// Accepted values are: "none", "consumer", "ip", "header", "cookie", "path", "query_arg", "uri_capture".
```

Comment:
I did not see `path` in accepted value of `hash_on` here in 3.0 docs preview: https://deploy-preview-4150--kongdocs.netlify.app/gateway/latest/admin-api/#upstream-object. Did I miss anything?